resources/qml/SidebarHeader.qml

```diff
@@ -196,16 +196,71 @@ Column
             }
 
             anchors.verticalCenter: parent.verticalCenter
-            width: parent.width * 0.45 - UM.Theme.getSize("default_margin").width
+            width: parent.width * 0.30
```

Comment:
I agree that it's silly to make space for a button that's not displayed on single-extrusion printers.